Crash when setting Adapter

When the RecyclerViewLayout is instantiated dynamically, using
```
mProgressRecyclerViewLayout = new ProgressRecyclerViewLayout(getContext());
LinearLayoutManager manager = new LinearLayoutManager(getContext());
manager.setOrientation(LinearLayoutManager.VERTICAL);
```

it crashes when setting the adapter with trace as
```
Caused by: java.lang.NullPointerException: 
Attempt to invoke virtual method 'void com.carrotcreative.recyclercore.widget.RecyclerCoreRecyclerView.setVisibility(int)' on a null object reference

at com.carrotcreative.recyclercore.widget.ProgressRecyclerViewLayout.resetViewVisibility(ProgressRecyclerViewLayout.java:364)

at com.carrotcreative.recyclercore.widget.ProgressRecyclerViewLayout.setAdapter(ProgressRecyclerViewLayout.java:113)
```

--- a/recycler-core/src/main/java/com/carrotcreative/recyclercore/widget/ProgressRecyclerViewLayout.java
+++ b/recycler-core/src/main/java/com/carrotcreative/recyclercore/widget/ProgressRecyclerViewLayout.java
@@ -57,12 +57,8 @@ public class ProgressRecyclerViewLayout extends RelativeLayout
         LayoutInflater inflater = (LayoutInflater) context
                 .getSystemService(Context.LAYOUT_INFLATER_SERVICE);
         inflater.inflate(R.layout.progress_recycler_view, this, true);
-    }
 
-    @Override
-    protected void onFinishInflate()
-    {
-        super.onFinishInflate();
+        // prepare the views
         findViews();
         setDefaultLayoutManager();
         setDefaultAdapter();

